### DISCUSSION TITLE
Teslascope figured out a way to get FSD status, can we?

### DISCUSSION THREAD

**Discussion by ygx0** [teslamate-org/teslamate#4197] [General]
![image](https://github.com/user-attachments/assets/6f1ce630-9f7c-42de-af31-acc1f7566c15)

**Comment by arcastro** [comment]
Looks like they're using the fleet API for this. To be supported by Teslamate, I think each individual instance owner would have to submit a separate application to Tesla to use the fleet API. 

https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api

<img src="https://github.com/user-attachments/assets/1a12a9bf-6c40-433b-8382-31b825be0324" width="450px"/>

**Comment by ygx0** [comment]
Looks like the Discovery plan is currently free. I would definitely be fine with setting up a custom app that Teslamate pulls from. However looks like daily request limitations might prevent live FSD data retrieval.
